app/BlueprintFramework/Libraries/ExtensionLibrary/Console/BlueprintConsoleLibrary.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+// Core file for the console library for Blueprint Extensions
+
+
+namespace Pterodactyl\BlueprintFramework\Libraries\ExtensionLibrary\Console;
+
+use Pterodactyl\Contracts\Repository\SettingsRepositoryInterface;
+
+class BlueprintConsoleLibrary
+{
+  // Construct core
+  public function __construct(
+    private SettingsRepositoryInterface $settings,
+  ) {
+  }
+
+  /**
+   * Fetch a record from the database.
+   * 
+   * @param string $table Database table
+   * @param string $record Database record
+   * @return mixed Database value
+   * 
+   * [BlueprintExtensionLibrary documentation](https://blueprint.zip/docs/?page=documentation/$blueprint)
+   */
+  public function dbGet($table, $record): mixed {
+    return $this->settings->get($table."::".$record);
+  }
+
+  /**
+   * Set a database record.
+   * 
+   * @param string $table Database table
+   * @param string $record Database record
+   * @param string $value Value to store
+   * 
+   * [BlueprintExtensionLibrary documentation](https://blueprint.zip/docs/?page=documentation/$blueprint)
+   */
+  public function dbSet($table, $record, $value) {
+    return $this->settings->set($table."::".$record, $value);
+  }
+
+  /**
+   * Delete/forget a database record.
+   * 
+   * @param string $table Database table
+   * @param string $record Database record
+   * 
+   * [BlueprintExtensionLibrary documentation](https://blueprint.zip/docs/?page=documentation/$blueprint)
+   */
+  public function dbForget($table, $record) {
+    return $this->settings->forget($table."::".$record);
+  }
+
+  /**
+   * Read and returns the content of a given file.
+   * 
+   * @param string $path Path to file
+   * @return string File contents
+   * @throws string Errors encountered by `cat` shell utility
+   * 
+   * [BlueprintExtensionLibrary documentation](https://blueprint.zip/docs/?page=documentation/$blueprint)
+   */
+  public function fileRead($path) {
+    return shell_exec("cat ".escapeshellarg($path).";");
+  }
+
+  /**
+   * Attempts to (non-recursively) create a file.
+   * 
+   * @param string $path File name/path
+   * @return string Empty string unless error
+   * @throws string Errors encountered by `touch` shell utility
+   * 
+   * [BlueprintExtensionLibrary documentation](https://blueprint.zip/docs/?page=documentation/$blueprint)
+   */
+  public function fileMake($path) {
+    return shell_exec("touch ".escapeshellarg($path).";");
```

Comment:
why the fuck is this a shell command
```suggestion
    $file = fopen($path, "w");
    fclose($file);
```